checkin/models.py

```diff
@@ -11,12 +18,28 @@ def __str__(self):
         return self.title
 
     def locale_title(self, language_code):
+        """
+        Finds the title of the skill in the given language
+
+        Supports Norwegian Bokmål and English, defaulting to English for all other language codes.
+
+        :param language_code: A string representing an ISO 639-1 language code
+        :return: Skill name in given language
+        """
         if language_code == "nb":
             return self.title
         return self.title_en
 
 
 class Profile(models.Model):
+    """
+    Profile model, storing status of last been at Makerverkstedet.
```

Comment:
"A model for user profiles, storing the status of the their last visit to Makerverkstedet"